app/build.gradle

```diff
@@ -3,47 +3,46 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 30
-    buildToolsVersion "30.0.1"
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
         applicationId "com.github.muellerma.coffee"
         minSdkVersion 21
         targetSdkVersion 30
         versionCode 7
         versionName "1.7"
-
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
 
     lintOptions {
         abortOnError false
     }
+
     compileOptions {
-        targetCompatibility 1.8
-        sourceCompatibility 1.8
+        encoding "UTF-8"
```

Comment:
Why is this needed?

Reply:
It helps keep files from possibly being loaded in the wrong encoding, particularly for Java 8.